x-pack/plugins/security_solution/public/management/components/management_empty_state.tsx

```diff
@@ -170,107 +174,210 @@ const EndpointsEmptyState = React.memo<{
+      () =>
+        policyItemsWithoutAgentPolicy.length > 0 && (
           <>
-            <EuiText color="subdued" size="m" grow={false}>
-              <FormattedMessage
-                id="xpack.securitySolution.endpoint.list.stepOne"
-                defaultMessage="Select from existing integrations. This can be changed later."
-              />
-            </EuiText>
-            <EuiSpacer size="xxl" />
-            <EuiSelectable
-              options={selectionOptions}
-              singleSelection="always"
-              isLoading={loading}
-              height={100}
-              listProps={{ bordered: true, singleSelection: true }}
-              onChange={handleSelectableOnChange}
-              data-test-subj="onboardingPolicySelect"
+            <EuiSpacer size="xl" />
+            <EuiCallOut
+              color="primary"
+              iconType="iInCircle"
+              title={i18n.translate(
+                'xpack.securitySolution.endpoint.list.notAddedIntegrations.title',
+                {
+                  defaultMessage: 'Integrations not added to an Agent Policy',
+                }
+              )}
+              data-test-subj="integrationsNotAddedToAgentPolicyCallout"
             >
-              {(list) => {
-                return loading ? (
-                  <EuiSelectableMessage>
-                    <FormattedMessage
-                      id="xpack.securitySolution.endpoint.list.loadingPolicies"
-                      defaultMessage="Loading integrations"
-                    />
-                  </EuiSelectableMessage>
-                ) : selectionOptions.length ? (
-                  list
-                ) : (
-                  <FormattedMessage
-                    id="xpack.securitySolution.endpoint.list.noPolicies"
-                    defaultMessage="There are no integrations."
-                  />
-                );
-              }}
-            </EuiSelectable>
+              <EuiSpacer size="s" />
+
+              <EuiText size="s" color="subdued">
+                <FormattedMessage
+                  id="xpack.securitySolution.endpoint.list.notAddedIntegrations.description"
+                  defaultMessage="There are some Elastic Defend integrations which are not added to an Agent Policy, therefore they cannot be deployed to an Agent."
+                />
+
+                <EuiSpacer size="s" />
+
+                <ul>
+                  {policyItemsWithoutAgentPolicy.map((policyItem) => (
+                    <li key={policyItem.id}>
+                      <EuiLink
+                        target="_blank"
+                        href={getAppUrl({
+                          appId: INTEGRATIONS_PLUGIN_ID,
+                          path: pagePathGetters.integration_policy_edit({
+                            packagePolicyId: policyItem.id,
+                          })[1],
+                        })}
+                      >
```

Comment:
```suggestion
                      <EuiLink
                        target="_blank"
                        href={getAppUrl({
                          appId: INTEGRATIONS_PLUGIN_ID,
                          path: pagePathGetters.integration_policy_edit({
                            packagePolicyId: policyItem.id,
                          })[1],
                        })}
                        data-test-subj="integrationWithoutAgentPolicyListItem"
                      >
```
The reason jest test is failing.

Reply:
i wonder where it got lost ヽ(。_°)ノ

thanks :D

Reply:
aha, here did 51c9ae272c1193235677eb51b306d30c197533d8
:')

Reply:
49c97b3658f9a18f9c201985756f228fdfc685e7